CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,41 @@
 
 import Foundation
 
-print("Hello, World!")
+let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
 
+func createLottoNumber() -> Set<Int> {
+	
+	var randomLotto: Set<Int> = []
+	
+	while randomLotto.count < 6 {
+		randomLotto.insert(Int.random(in: 1...45))
+	}
+	
+	return randomLotto
+}
+
+func checkLottoNumbers(myLottoNumbers: [Int]) -> Void {
+	var correctNumbers = Array<Int>()
+	let lottoNumbers = createLottoNumber()
+	var lottoString = ""
+
+	for number in myLottoNumbers {
+		if(lottoNumbers.contains(number)) {
```

Comment:
```suggestion
		if lottoNumbers.contains(number) {
```
위와 같이 쓰는 것이 적절한 컨벤션일 것 같습니다.